cromwellApiClient/src/test/scala/cromwell/api/model/CromwellQueryResultJsonFormatterSpec.scala

```diff
@@ -21,21 +33,20 @@ class CromwellQueryResultJsonFormatterSpec extends FlatSpec with Matchers {
                        |      "name": "switcheroo",
                        |      "id": "bee51f36-396d-4e22-8a81-33dedff66bf6",
                        |      "status": "Failed",
-                       |      "end": "2017-07-24T14:44:34.010-04:00",
-                       |      "start": "2017-07-24T14:44:33.227-04:00"
+                       |      "end": "2017-07-24T14:44:34.010Z",
+                       |      "start": "2017-07-24T14:44:33.227Z"
```

Comment:
Weird but possibly correct? The time zone has changed by 4 hours but the hour field remains the same?

Reply:
I changed this by hand just and just turned `-04:00` to `Z`. I think it's just the GitHub word-highlighting that also selected the milliseconds.